cmd/aperturectl/cmd/utils/controller.go

```diff
@@ -92,15 +96,41 @@ func (c *ControllerConn) InitFlags(flags *flag.FlagSet) {
 		"",
 		"FluxNinja ARC API Key to be used when using Cloud Controller",
 	)
+	flags.StringVar(
+		&c.config,
+		"config",
+		"",
+		"Path to the Aperture config file",
+	)
 }
 
 // PreRunE verifies flags (optionally loading kubeconfig) and should be run at PreRunE stage.
 func (c *ControllerConn) PreRunE(_ *cobra.Command, _ []string) error {
-	if c.controllerAddr == "" && !c.kube {
-		log.Info().Msg("Neither --controller nor --kube flag is set. Assuming --kube=true.")
+	env := os.Getenv(configEnv)
+	if env == "" {
+		homeDir, err := os.UserHomeDir()
+		if err == nil {
+			env = filepath.Join(homeDir, ".aperturectl", "config")
+			if _, err := os.Stat(env); err != nil {
+				env = ""
+			}
+		}
+	}
+
+	updatedConfig := c.config
```

Comment:
Nit: The `updatedConfig` name is very confusing. Can we just avoid using different variable and use `c.config`?

If you want to keep this check
```
	if c.config != "" && c.apiKey != "" {
		return errors.New("--api-key cannot be used with --config")
	}
```

it can be moved above.

Or perhaps, move all the config-path resolution down to line 111 (just before `if c.kube`). That way, flag verification will happen before we check env variables, which make sense to me.